### PR TITLE
I21 - junit tests

### PR DESCRIPTION
**Pull request by hillalex**
First merge https://github.com/mrc-ide/google-form-post/pull/1

### CODE REVIEW

File: src/test/java/org/imperial/dide/GoogleFormPostTests.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class GoogleFormPostTests {
+
+    private HttpRequestManager httpRequestManagerMock = mock(HttpRequestManager.class);
+    private CallbackContext callbackContextMock = mock(CallbackContext.class);
+
+    @Test
+    public void ReturnsSuccessIfFormPostSuccessful() throws JSONException, IOException {
```
Comment (MartinEden) [inline src/test/java/org/imperial/dide/GoogleFormPostTests.java]: Normal to start Java methods with a lower case letter.

File: package.json [hunk 2]
```diff
@@ -1,19 +1,30 @@
 {
   "name": "google-form-post",
   "version": "1.0.0",
-  "description": "Submits a response to a google form",
+  "description": "Cordova plugin that submits a response to a google form",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha --compilers js:babel-core/register www/test/*.js"
```
Comment (MartinEden) [inline package.json]: Is there anything in the js test folder?

**Review by MartinEden — CHANGES_REQUESTED** [review]
Looks really nice. Could you add to the README how to run the tests?

File: package.json (hunk 2 above)
Comment (hillalex) [reply package.json]: Ah, no. I added a test file and then realised the js was so simple it couldn't really be tested. I'll remove the js testing machinery.

**Review by MartinEden — APPROVED** [review]

**Pull request merged**